extensions/amp-ad-exit/amp-ad-exit.md

```diff
@@ -0,0 +1,246 @@
+
+1. navigator.sendBeacon, if available
+2. image request
+
+You can override this behavior with a "transport" object on the config:
+
+``` javascript
+{
+  "targets": { ... },
+  "filters": { ... },
+  "transport": {
+    "beacon": false,
+  }
+}
+```
+
+## <a name="variable-substitution"></a> Variable Substitution
+
+URL variable substitution works like standard [AMP variable substitution](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md) with
+custom variables and a limited set of platform variables. Variable substitution
+applies to navigation URLs and click tracking URLs.
+
+### Platform variables
+
+| Name | Value |
+| ---- | ----- |
+| RANDOM | A random float. See [RANDOM](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#random). |
+| CLICK_X | The `x` coordinate of the click in the viewport. |
+| CLICK_Y | The `y` coordinate of the click in the viewport. |
+
+### Custom variables
+Custom variables must begin with an underscore. Define variables in the
+config alongside the navigation target. The default value can be overridden
+in the `exit` action invocation:
+
+```html
+<amp-ad-exit id="exit-api"><script type="application/json">
+{
+  "targets": {
+    "product": {
+      "finalUrl": "http://example.com/?page=_PRODUCT",
+      "vars": {
+        "_PRODUCT": {
+          "defaultValue": "none"
+        }
+      }
+    }
+  }
+}
+</script></amp-ad-exit>
+<a on="tap:exit-api.exit(target='product', _PRODUCT='shoes')">buy shoes</a>
+<a on="tap:exit-api.exit(target='product', _PRODUCT='hats')">buy hats</a>
+```
+
+WARNING: Be careful with your variable names. Substitution works by simple
+string replacement. *Any* occurence of the variable in the URL will be
+replaced. For example, if you define a custom variable named "_b" with value
+"foo", `/?a_b_c=_b` will become `/?afoo_c=foo`.
+
+## <a name="exit-action"></a> `exit` action
+
+amp-ad-exit exposes an `exit` action that other elements will reference in `on="tap:..."` attributes. The action accepts a "target" string parameter that must match a named `NavigationTarget` in the `ExitConfig`. Custom variables beggining with an underscore can also be passed in.
+
+| Parameter name      | Parameter value type      | Meaning                    |
+| ------------------- | ------------------------- | -------------------------- |
+| `target`            | `string`                  | The name of a  `NavigationTarget` in the `ExitConfig` |
+| `_[a-zA-Z0-9_-]+` | `string\|boolean\|number` | Replace the URL parameter with this name and value into the final and tracking URLs. |
+
+## <a name="config"></a> Configuration spec
+See the `AmpAdExitConfig` typedef in config.js.
+
+## Attributes
+
+amp-ad-exit needs an `id` to be referenced by tappable elements.
+
+## Validation
+See [amp-ad-exit rules](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-exit/0.1/validator-amp-ad-exit.protoascii) in the AMP validator specification.
```

Comment:
please also mention that this extension is only available in `amp4ads`
@Gregable FYI, we need validator support for this.